README.md

```diff
@@ -6,19 +6,36 @@ leaflet-blurred-location provides friendly interfaces for:
 * Marking your location on a map
 * Tagging additional location with marked places
 
+## Setting up leaflet-blurred-location
+
+To set up the library first clone this repo to your local after that run 'npm install' to install all the neccessary packages required. Then open examples/index.html to look at the preview of the library.
+
+## Creating a map object
+
+To create a new object just call the constructor 'BlurredLocation' as shown in the following example:
+
+'// this "constructs" an instance of the library:
+var object = new BlurredLocation({
+  lat: 41.01,
+  lon: -85.66
+});
+
+object.getLat(); // should return 41.01
+object.getLon(); // should return -85.66'
```

Comment:
Looks like a trailing `'` here, and on line 17 - mind tweaking this? Thanks!

This all looks great. I think it's great to begin with a good README file. Perhaps a line at the top like:

`**This is a working draft; the project will be working towards an initial v0.0.1 release in coming weeks**` -- that way people don't try to follow the steps just yet!